public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -463,6 +463,28 @@ export class DashboardMigrator {
       });
     }
 
+    if (oldVersion < 21) {
+      const updateLinks = (link: DataLink) => {
+        return {
+          ...link,
+          url: link.url.replace(/__series.labels/g, '__field.labels'),
```

Comment:
I think this should be safer with `/${__series.labels}/` so we do not replace something that is not a variable. Or is there multiple syntaxes for this? I know that there are for dashboard variables but thought in datalink only `${}` is allowed.

Reply:
@dprokop  ?